### PR TITLE
:pencil: add quarter circle isoparametric validation

### PR DESCRIPTION
**Pull request by jgiven100**
**Describe the PR**

* Added the Mohr-Coulomb infinite plate validation for isoparametric elements (results for both a 10m and 30m model).
* Tried to match the format of existing validations as closely as possible. 

**Related Issues/PRs**
n/a

**Additional context**
n/a

### CODE REVIEW

File: user/validation/explicit/plate-hole-iso/plate-hole-iso.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+## Analytical Solution
+
+The closed form solution for this problem is from Salencon [1]. The normalized form of the analytical solution is from [2]. 
+
+### Elastic-plastic boundary
+
+The yield zone radius ($R_0$) is
+
+$$\frac{R_0}{a} = \left(\frac{2}{K_p+1}\frac{1+\frac{q}{P_0}k_p}{\frac{P_i}{P_0}+\frac{q}{P_0}k_p}\right)^{k_p}$$
```
Comment (bodhinandach) [inline user/validation/explicit/plate-hole-iso/plate-hole-iso.md:17]: Thanks for adding the example! @jgiven100 Will mark down convert the equation written using latex format?

Comment (jgiven100) [reply user/validation/explicit/plate-hole-iso/plate-hole-iso.md:17]: @bodhinandach Yes equations will look good using Latex in an `.md` doc. Although I'm not sure if that for all github pages or a particular setting on our cb-geo site

**Review by bodhinandach — APPROVED** [review]

File: user/validation/explicit/plate-hole-iso/plate-hole-iso.md (hunk 1 above)
Comment (kks32) [reply user/validation/explicit/plate-hole-iso/plate-hole-iso.md:17]: https://deploy-preview-50--mpm.netlify.app/#/user/validation/explicit/plate-hole-iso/plate-hole-iso

**Review by kks32 — APPROVED** [review]
Thanks for the PR @jgiven100 could you please add it to our mpm-benchmarks file too?

**Comment by jgiven100** [comment]
@kks32 Should I just replace the existing files or make a new directory? Here is the existing mpm-benchmarks isoparametric format
https://github.com/cb-geo/mpm-benchmarks/tree/develop/2d/plate_with_hole/isoparametric

**Comment by kks32** [comment]
> @kks32 Should I just replace the existing files or make a new directory? Here is the existing mpm-benchmarks isoparametric format
> https://github.com/cb-geo/mpm-benchmarks/tree/develop/2d/plate_with_hole/isoparametric

@jgiven100 replace please

**Pull request merged**